web/test/pages/profile.test.tsx

```diff
@@ -1727,6 +1727,29 @@ describe("profile", () => {
     });
   });
 
+  describe("Information Section", () => {
+    describe("business structure field", () => {
```

Comment:
we still call this legal structure in the code, and we should continue to use only one terminology for it